wagtail/contrib/forms/views.py

```diff
@@ -318,5 +329,7 @@ def get_context_data(self, **kwargs):
                     "data_rows": data_rows,
                 }
             )
-
+        context["next_url"] = (
+            self.request.path + "?" + self.request.META["QUERY_STRING"]
+        )
```

Comment:
I think we can use [`request.get_full_path()`](https://docs.djangoproject.com/en/stable/ref/request-response/#django.http.HttpRequest.get_full_path) instead.